webcompat/static/js/lib/wizard/steps.js

```diff
@@ -0,0 +1,27 @@
+import step1 from "./step1.js";
+import step2 from "./step2.js";
+import step3 from "./step3.js";
+import step4 from "./step4.js";
+import step5 from "./step5.js";
+import step6 from "./step6.js";
+import step7 from "./step7.js";
+import step8 from "./step8.js";
+import step9 from "./step9.js";
+import step10 from "./step10.js";
```

Comment:
Do we run the risk here of having to rename everything the day we need to remove, add steps or change the order? In historical discussions, we would need to do the mental translation so `step1-2020 is our step3-2021`. z

maybe we could find a system which based on names of what the step is doing instead of a number. If we we want to highlight the continuity in between the module, we could maybe mention in a comment the step orders.

For example step1 seems to be about url input, so maybe we should call it instead 

```js
import step_url from "./step_url.js" 
```

or what ever naming convention is the good way in JavaScript.

Reply:
Yeah, good point :) I've noticed sometimes when I open a file, can't really tell which step this is until I find it in html 😳  I'll rename them while working on the next issue